services/pingpong/pingpong-uptime.tester.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const Joi = require('joi')
+const t = (module.exports = require('../tester').createServiceTester())
+
+// Example matches: 87.33%, 100%, 51%
+const isCorrectMessage = Joi.string().pattern(
+  /\b(?<!\.)(?!0+(?:\.0+)?%)(?:\d|[1-9]\d|100)(?:(?<!100)\.\d+)?%/
+)
```

Comment:
I feel like we should be able to replace this with our `isPercentage` helper from `test-validators` and avoid having another  regex to maintain. Could you give that a try, or explain why we need a separate one here?